mappings/net/minecraft/world/ChunkUpdateState.mapping

```diff
@@ -6,5 +6,5 @@ CLASS cic net/minecraft/world/ChunkUpdateState
 	METHOD a getAll ()Lit/unimi/dsi/fastutil/longs/LongSet;
 	METHOD a add (J)V
 	METHOD b contains (J)Z
-	METHOD c isRemaing (J)Z
+	METHOD c isRemaining (J)Z
```

Comment:
Sounds a bit vague to me

Reply:
just a typo fix. this should be block updates for unloaded chunks (like a task from a new vanilla command)